### ISSUE TITLE
Unable to connect Presto in v0.34.0

### ISSUE THREAD

**Issue by prakharjain-vogo** [metabase/metabase#11598]
**Describe the bug**
We are not able to connect to PrestoDB from Metabase v0.34.0. We were able to connect it in v0.33.6. We get following error in UI (Removed Hostname and Username from image):
![image](https://user-images.githubusercontent.com/55583247/71504775-b54fb680-28a0-11ea-923a-0491338b1af3.png)

**Logs**
Please find below server logs:
```
12-26 16:31:47 [1mERROR driver.util[0m :: Database connection error

16:31:47
java.lang.AssertionError: Assert failed: ((some-fn keyword? string?) driver)

16:31:47
at metabase.driver$the_driver.invokeStatic(driver.clj:97)

16:31:47
at metabase.driver$the_driver.invoke(driver.clj:97)

16:31:47
at metabase.driver$the_initialized_driver.invokeStatic(driver.clj:145)

16:31:47
at metabase.driver$the_initialized_driver.invoke(driver.clj:142)

16:31:47
at metabase.driver$dispatch_on_initialized_driver.invokeStatic(driver.clj:154)

16:31:47
at metabase.driver$dispatch_on_initialized_driver.doInvoke(driver.clj:149)

16:31:47
at clojure.lang.RestFn.invoke(RestFn.java:410)

16:31:47
at metabase.driver$fn__21027$fn__21028.invoke(driver.clj:399)

16:31:47
at clojure.lang.MultiFn.invoke(MultiFn.java:233)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invokeStatic(timezone.clj:45)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invoke(timezone.clj:38)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invokeStatic(timezone.clj:42)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invoke(timezone.clj:38)

16:31:47
at metabase.driver.presto$details__GT_request.invokeStatic(presto.clj:54)

16:31:47
at metabase.driver.presto$details__GT_request.invoke(presto.clj:49)

16:31:47
at metabase.driver.presto$execute_presto_query_BANG_$fn__318.invoke(presto.clj:101)

16:31:47
at metabase.util.ssh$with_ssh_tunnel_STAR_.invokeStatic(ssh.clj:106)

16:31:47
at metabase.util.ssh$with_ssh_tunnel_STAR_.invoke(ssh.clj:91)

16:31:47
at metabase.driver.presto$execute_presto_query_BANG_.invokeStatic(presto.clj:98)

16:31:47
at metabase.driver.presto$execute_presto_query_BANG_.invoke(presto.clj:94)

16:31:47
at metabase.driver.presto$eval336$can_connect_QMARK___335__347$fn__351.invoke(presto.clj:138)

16:31:47
at metabase.driver.presto$eval336$can_connect_QMARK___335__347.invoke(presto.clj:136)

16:31:47
at clojure.lang.AFn.applyToHelper(AFn.java:156)

16:31:47
at clojure.lang.AFn.applyTo(AFn.java:144)

16:31:47
at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)

16:31:47
at clojure.lang.RestFn.invoke(RestFn.java:421)

16:31:47
at clojure.lang.MultiFn.invoke(MultiFn.java:234)

16:31:47
at metabase.driver.util$can_connect_with_details_QMARK_$fn__21110.invoke(util.clj:32)

16:31:47
at metabase.util$do_with_timeout$fn__13131.invoke(util.clj:302)

16:31:47
at clojure.core$binding_conveyor_fn$fn__5754.invoke(core.clj:2030)

16:31:47
at clojure.lang.AFn.call(AFn.java:18)

16:31:47
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)

16:31:47
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)

16:31:47
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)

16:31:47
at java.base/java.lang.Thread.run(Unknown Source)

16:31:47
12-26 16:31:47 [1mERROR driver.util[0m :: Database connection error

16:31:47
java.lang.AssertionError: Assert failed: ((some-fn keyword? string?) driver)

16:31:47
at metabase.driver$the_driver.invokeStatic(driver.clj:97)

16:31:47
at metabase.driver$the_driver.invoke(driver.clj:97)

16:31:47
at metabase.driver$the_initialized_driver.invokeStatic(driver.clj:145)

16:31:47
at metabase.driver$the_initialized_driver.invoke(driver.clj:142)

16:31:47
at metabase.driver$dispatch_on_initialized_driver.invokeStatic(driver.clj:154)

16:31:47
at metabase.driver$dispatch_on_initialized_driver.doInvoke(driver.clj:149)

16:31:47
at clojure.lang.RestFn.invoke(RestFn.java:410)

16:31:47
at metabase.driver$fn__21027$fn__21028.invoke(driver.clj:399)

16:31:47
at clojure.lang.MultiFn.invoke(MultiFn.java:233)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invokeStatic(timezone.clj:45)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invoke(timezone.clj:38)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invokeStatic(timezone.clj:42)

16:31:47
at metabase.query_processor.timezone$report_timezone_id_if_supported.invoke(timezone.clj:38)

16:31:47
at metabase.driver.presto$details__GT_request.invokeStatic(presto.clj:54)

16:31:47
at metabase.driver.presto$details__GT_request.invoke(presto.clj:49)

16:31:47
at metabase.driver.presto$execute_presto_query_BANG_$fn__318.invoke(presto.clj:101)

16:31:47
at metabase.util.ssh$with_ssh_tunnel_STAR_.invokeStatic(ssh.clj:106)

16:31:47
at metabase.util.ssh$with_ssh_tunnel_STAR_.invoke(ssh.clj:91)

16:31:47
at metabase.driver.presto$execute_presto_query_BANG_.invokeStatic(presto.clj:98)

16:31:47
at metabase.driver.presto$execute_presto_query_BANG_.invoke(presto.clj:94)

16:31:47
at metabase.driver.presto$eval336$can_connect_QMARK___335__347$fn__351.invoke(presto.clj:138)

16:31:47
at metabase.driver.presto$eval336$can_connect_QMARK___335__347.invoke(presto.clj:136)

16:31:47
at clojure.lang.AFn.applyToHelper(AFn.java:156)

16:31:47
at clojure.lang.AFn.applyTo(AFn.java:144)

16:31:47
at clojure.lang.AFunction$1.doInvoke(AFunction.java:31)

16:31:47
at clojure.lang.RestFn.invoke(RestFn.java:421)

16:31:47
at clojure.lang.MultiFn.invoke(MultiFn.java:234)

16:31:47
at metabase.driver.util$can_connect_with_details_QMARK_$fn__21110.invoke(util.clj:32)

16:31:47
at metabase.util$do_with_timeout$fn__13131.invoke(util.clj:302)

16:31:47
at clojure.core$binding_conveyor_fn$fn__5754.invoke(core.clj:2030)

16:31:47
at clojure.lang.AFn.call(AFn.java:18)

16:31:47
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)

16:31:47
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)

16:31:47
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)

16:31:47
at java.base/java.lang.Thread.run(Unknown Source)

16:31:47
12-26 16:31:47 [1mDEBUG middleware.log[0m :: [32mPOST /api/database/validate 200 7.5 ms (0 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (71 total active threads) Queries in flight: 0[0m

16:31:57
12-26 16:31:57 [1mDEBUG middleware.log[0m :: [32mGET /api/public/card/61353e2d-f20b-4a05-9a60-a9d69b8e9f9d 200 2.2 ms (1 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (71 total active threads) Queries in flight: 0[0m

16:31:57
12-26 16:31:57 [1mDEBUG middleware.log[0m :: [31mGET /api/user/current 401 168.5 µs (0 DB calls)

16:31:57
"Unauthenticated"

16:31:57
[0m

16:32:01
12-26 16:32:01 [1mDEBUG middleware.log[0m :: [31mGET /api/user/current 401 153.7 µs (0 DB calls)

16:32:01
"Unauthenticated"
```

**Expected behavior**
Should get connected

**Screenshots**

**Information about your Metabase Installation:**

```json
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0.3 Safari/605.1.15",
    "vendor": "Apple Computer, Inc."
  },
  "system-info": {
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.3+7",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.3",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.3+7",
    "os.name": "Linux",
    "os.version": "4.4.0-1083-aws",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "postgres",
      "redshift",
      "mongo",
      "presto"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "run-mode": "prod",
    "version": {
      "date": "2019-12-19",
      "tag": "v0.34.0",
      "branch": "release-0.34.x",
      "hash": "1335215"
    },
    "settings": {
      "report-timezone": "Asia/Calcutta"
    }
  }
}
```

**Severity**
It is VERY SEVERE issue, blocking our main PrestoDB metabase usage.

**Additional context**
Something from v0.34.0 got this up. Please fix it asap.

**Comment by mayconbordin** [comment]
I'm also facing this issue.

**Comment by prakharjain-vogo** [comment]
@mayconbordin did you got any solution?

**Comment by EBallou** [comment]
ditto on issue... may we get ETA since now we need to consider rollback.

**Comment by flamber** [comment]
I would not expect any new release until Monday at a minimum, which might not include a fix for this, since it still needs to be investigated, so I would recommend a downgrade to 0.33.7.3 until then.
Remember to backup before downgrading/upgrading.

**Comment by dacort** [comment]
Just a note that I was able to reproduce this locally with Metabase v0.34.0 and PrestoSQL 327.

**Comment by camsaul** [comment]
Fixed by #11687
